synadm/cli/user.py

```diff
@@ -554,3 +554,24 @@ def user_login_cmd(helper, user_id, expire_days, expire, expire_ts,
             raise SystemExit(1)
         else:
             helper.output(user_login)
+
+@user.command(name="shadow-ban")
+@click.argument("user_id", type=str)
+@click.option(
+    "-u", "--unban", is_flag=True, default=False, show_default=False,
+    help="Unban the specified user"
+)
+@click.pass_obj
+def user_shadow_ban(helper, user_id, unban):
+    """Shadow-ban or unban a user.
+
+    Useful for moderating malicious or egregiously abusive users.
+
+    A shadow-banned user will not receive any notification, but still will be
+    unable to contact anyone on the server. Use this as a tool of last resort since it
+    may lead to confusing or broken behaviour for the client. 
```

Comment:
Please keep the lines 79 characters max. Don't know what editor you are using but with something `vim`'ish the quickest way would be marking the lines and pressing `gq`

Reply:
Also there is a redundant space after client.